src/components/Settings.js

```diff
@@ -62,6 +68,18 @@ export const Settings = (props) => {
         checked={props.settings.hiddenSSID}
         onChange={() => props.onHiddenSSIDChange(!props.settings.hiddenSSID)}
       />
+      <TextInputField
+        type="number"
+        width={300}
+        label={t('cards.additional')}
+        value={props.settings.additionalCards}
+        onChange={(e) => props.onAdditionalCardsChange(e.target.value)}
+      />
+      <Checkbox
```

Comment:
I think this should move up and live next to the other checkboxes, directly under "Hidden SSID":
<img width="610" alt="image" src="https://github.com/bndw/wifi-card/assets/4248167/d013622d-64d3-4d22-89c4-e682c5ed5b03">

Reply:
Sure! So the change would be: put "hide tip" as the last option before "encryption" and "additional cards" to the bottom,  right after all the encryption options, right?

Reply:
Correct, immediately following [this checkbox](https://github.com/bndw/wifi-card/blob/master/src/components/Settings.js#L60-L64) on line 65.

Reply:
Done in https://github.com/bndw/wifi-card/pull/275/commits/d00f863397cd0ed567b3720e329d39d411ef4feb